libr/util/num.c

```diff
@@ -7,32 +7,6 @@
 #include <r_util.h>
 #define R_NUM_USE_CALC 1
 
-R_API ut16 r_num_ntohs (ut16 foo) {
-#if LIL_ENDIAN
-	ut8 *p = (ut8*)&foo;
-	foo = p[1] | p[0]<<8;
-#endif
-	return foo;
-}
-
-#define __htonq(x) (\
```

Comment:
ok, hopefully lllvm/gcc will optimize that for something more native :3 but ok  to remove all this. maybe we should also remove the endian check in the configure script too. when we are 100% sure there's no endian-specific code in r2